packages/manager/apps/pci-savings-plan/src/types/api.type.ts

```diff
@@ -65,3 +67,12 @@ export interface SavingsPlanService {
   startDate: string;
   status: SavingsPlanStatus;
 }
+
+export type MutationInfo<Data = void> = {
```

Comment:
Prefer using generics: 
```suggestion
export type MutationInfo<T = void> = {
```